tests/testthat/test-data_home_sidebar.R

```diff
@@ -2,6 +2,10 @@ test_that("data_home_sidebar() works by default", {
   pkg <- test_path("assets/sidebar")
   pkg <- as_pkgdown(pkg)
   expect_snapshot(cat(data_home_sidebar(pkg)))
+
+  pkg <- test_path("assets/sidebar-comment")
+  pkg <- as_pkgdown(pkg)
+  expect_snapshot(cat(data_home_sidebar(pkg)))
```

Comment:
Can you remove all other components from the sidebar apart from the authors? That'll make the snapshot a little crisper.